src/typesdoc.cpp

```diff
@@ -104,11 +104,33 @@ TypesDoc::TypesDoc()
         "800x600+800+0 or 400x200-10+30"
     );
 
+    regex_.init(Type::REGEX);
+    regex_.setDoc(
+        "A POSIX extended regular expression. Note that when passing "
+        "a regex on the command line, additional quoting can be "
+        "necessary. "
+        "For explanations and examples, see section 9.4.6 "
+        "of the documentation: "
+        "https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap09.html#tag_09_04_06"
+    );
+
     string_.init(Type::STRING);
     string_.setDoc("Type representing normal text.");
 
     uint_.init(Type::ULONG);
     uint_.setDoc("Type representing unsigned (i.e. non-negative) integers");
+
+    windowid_.init(Type::WINDOW);
+    windowid_.setDoc(
+        "The window id is the number of a window. This can be a managed window "
+        "(i.e. client) or an unmanaged window (e.g. a panel, a menu, "
+        "or a desktop window).\n"
+        "The default format is 0xHEX where HEX is a hexadecimal number "
+        "(digits 0-9 and a-z) but it can also be specified in the decimal "
```

Comment:
This should be a-f

Reply:
Oh, absolutely! Thanks, it's fixed now :)

Reply:
Thanks!